test-utils/src/lib.rs

```diff
@@ -293,3 +297,34 @@ pub fn build_expected_hash(values: &[u64]) -> [Felt; 4] {
 
     expected
 }
+
+/// Mutates a seed and generates a word deterministically
+#[cfg(not(target_family = "wasm"))]
+pub fn seeded_word(seed: &mut [u8; 32]) -> Word {
+    [
+        seeded_element(seed),
+        seeded_element(seed),
+        seeded_element(seed),
+        seeded_element(seed),
+    ]
+}
+
+/// Mutates a seed and generates an element deterministically
+#[cfg(not(target_family = "wasm"))]
+pub fn seeded_element(seed: &mut [u8; 32]) -> Felt {
+    *seed = rand::prng_array(*seed);
+    let bytes = rand::prng_array(*seed);
+    let num = u64::from_le_bytes(bytes);
+    Felt::new(num)
+}
```

Comment:
Also, not sure we need this one either. If we need a pseudo-random `Felt` we can do:
```Rust
let x = rand::prng_array::<Felt, 1>(seed)[0];
```
Would be nice of course to have `rand::prng_value()` function - but we can do it in a different PR.

Reply:
As mentioned on the comment above, the function is a helper to mutate the seed + generate the value.

Let's keep the discussion centralized here: https://github.com/0xPolygonMiden/miden-vm/pull/808#discussion_r1160941348